src/components/Principle.svelte

```diff
@@ -58,9 +58,13 @@
     </LinkToGuidance>
   </p>
 
+  {#if $evaluation['meta']['principleApplicability'][principle.num]}
```

Comment:
now we can do `if` / `else` based on the applicability for a specific principle.